roles/umount_remove_lv/library/lvg_fix.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# flake8: noqa
+
+# Copyright: (c) 2013, Alexander Bulimov <lazywolf0@gmail.com>
+# Based on lvol module by Jeroen Hoekx <jeroen.hoekx@dsquare.be>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Uh uh uh, that's a no-go...

Reply:
https://github.com/ansible/ansible/pull/47620/files, seems to be fixed upstream. Guess we have to wait for a release and jump to it ?

Reply:
(seems good on ansible 2.6.4)